src/components/FiltersAndSorting/FiltersAndSorting.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+import classes from './FiltersAndSorting.scss';
+
+const FiltersAndSorting = () => {
+    const slidersIconClasses = classNames(
+        'fa fa-sliders fa-lg',
```

Comment:
Would be a little better to split this long string into separate array items:
```suggestion
        'fa', 'fa-sliders', 'fa-lg',
```